### PR TITLE
Add definitions for Leap 15.3 poo#76174

### PR DESCRIPTION
**Pull request by lkocman**
* Contains both .in and .xml files, that seems to be
  the habbit here
* armv7 remains in ports rest arches have simply copied repos from the
  Default one. I wonder whether I should reference this file from all of
  Server* files then. Keeping existing structure for consistency
* No longer supporting armv6

### CODE REVIEW

**Comment by ggardet** [comment]
LGTM :)

File: YaST/Repos/generate_ports.sh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # TODO: handle non-OSS repo removal automatically
 
 additionnal_archs="aarch64 armv7hl armv6hl ppc"
```
Comment (nilxam) [inline YaST/Repos/generate_ports.sh:5]: this line can not be true for 15.3, aarch64 and powerpc has the same online repo like x86_64

File: YaST/Repos/openSUSE_aarch64_Leap_15.3_Servers.xml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nilxam) [inline YaST/Repos/openSUSE_aarch64_Leap_15.3_Servers.xml:1]: we don't need this file, besides armv7, other supported archs should've the same url(no ports), and we don't change the external link in the skelcd-control(again, besides armv7)

https://build.opensuse.org/package/view_file/openSUSE:Leap:15.3/skelcd-control-openSUSE/skelcd-control-openSUSE.spec?expand=1

File: YaST/Repos/openSUSE_aarch64_Leap_15.3_Servers.xml.in [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
```
Comment (nilxam) [inline YaST/Repos/openSUSE_aarch64_Leap_15.3_Servers.xml.in:1]: we don't need this file, besides armv7, other supported archs should've the same url(no ports), and we don't change the external link in the skelcd-control(again, besides armv7)

https://build.opensuse.org/package/view_file/openSUSE:Leap:15.3/skelcd-control-openSUSE/skelcd-control-openSUSE.spec?expand=1

File: YaST/Repos/openSUSE_ppc_Leap_15.3_Servers.xml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nilxam) [inline YaST/Repos/openSUSE_ppc_Leap_15.3_Servers.xml:1]: we don't need this file, besides armv7, other supported archs should've the same url(no ports), and we don't change the external link in the skelcd-control(again, besides armv7)

https://build.opensuse.org/package/view_file/openSUSE:Leap:15.3/skelcd-control-openSUSE/skelcd-control-openSUSE.spec?expand=1

File: YaST/Repos/openSUSE_ppc_Leap_15.3_Servers.xml.in [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
```
Comment (nilxam) [inline YaST/Repos/openSUSE_ppc_Leap_15.3_Servers.xml.in:1]: we don't need this file, besides armv7, other supported archs should've the same url(no ports), and we don't change the external link in the skelcd-control(again, besides armv7)

https://build.opensuse.org/package/view_file/openSUSE:Leap:15.3/skelcd-control-openSUSE/skelcd-control-openSUSE.spec?expand=1

**Review by nilxam — CHANGES_REQUESTED** [review]
I think these aarch64, ppc's `.xml` or `.xml.in` just unnecessary, as you can seen the external link's url all the same, of course `armv7` is different.

**Comment by nilxam** [comment]
how about https://github.com/openSUSE/download.o.o/pull/33

File: YaST/Repos/generate_ports.sh (hunk 1 above)
Comment (lkocman) [reply YaST/Repos/generate_ports.sh:5]: well I didn't use this script :-) but I'll fix it. Thank you!

**Comment by lkocman** [comment]
Closed in favor of #33

**Pull request closed without merging**